docs/develop/sdk/1-dev-setup.md

```diff
@@ -43,8 +44,8 @@ After all these dependencies are added, we will add a simple script that prints
 Copy the above code into index.ts.
 You can know execute the script:
 
-```bash npm2yarn
-npm exec ts-node index.ts
+```bash
+ts-node index.ts
```

Comment:
If you are not using `npm2yarn` anymore, then it is sufficient to keep the same installation instructions as before, and just run `yarn ts-node index.ts`.

Reply:
Meaning that there's no need to install ts-node globally.

Reply:
yeah i only use yarn now
[075b539](https://github.com/KILTprotocol/docs/pull/81/commits/075b5393079b540aad3e80e51cca5183d2b02e40)